packages/glimmer-test-helpers/lib/environment.ts

```diff
@@ -507,6 +519,10 @@ class EmberishCurlyComponentManager implements ComponentManager<EmberishCurlyCom
     }
   }
 
+  didRenderLayout(component: EmberishCurlyComponent, bounds: Bounds) {
+    component.bounds = bounds;
```

Comment:
Also shaping issues with this, I'm assuming this is for getClientRect(view) so we just need to loosly  associate a bounds